repo-transfer.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Transfer a repository to the W3C GitHub organization</title>
+    <link rel="stylesheet" href="css/wgio.css">
+    <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico">
+  </head>
+  <body>
+    <header>
+      <h1>Transfer a repository to the W3C GitHub organization</h1>
+    </header>
+<nav>
+  <a href="/">Home</a>
+  •
+  <a href="https://github.com/w3c/">Repositories</a>
+  •
+  <a href="https://help.github.com/">GitHub Help</a>
+</nav>
+
+<main>
+  <p>This document describes how to transfer a repository to the W3C GitHub organization.</p>
+  <p>General information about repository transfer on GitHub is available <a href="https://docs.github.com/en/github/administering-a-repository/transferring-a-repository">here</a>.</p>
+  <ol>
+    <li>Make sure the <a href="https://github.com/w3c/">W3C GitHub organization</a> doesn't already have a repository with the same name. If the name is already taken, the owner will need to rename the repository first
+    <li>The owner of the repository requests the transfer from the repository settings:
+      <ul>
+        <li>if the owner is a W3C staff, they can transfer the repository directly to the W3C organization
+        <li>if the owner is an individual that cannot create new repository under the W3C organization, they need to transfer the repository to a W3C staff who will then transfer to the organization
+        <li>if the owner is another organization, a W3C staff first needs to join the organization with enough privileges to make the transfer and that staff will issue the transfer to the W3C organization
+      </ul>
+      <img src="img/github-transfer.png" title="GitHub repository transfer"/>
+  </ol>
+
+  <p>Once the tansfer is done, the next steps vary depending on the nature of the repository (specifications development or tooling).</p>
+  <ul>
+    <li>if the repository is about specifications development, it needs to be tracked by the repository manager. To do so, someone with admin rights on the repository needs to <a href="https://labs.w3.org/repo-manager/repo/import">import it to the repository manager</a> so contributions can be checked. The repository manager will offer to add some files (<code>w3c.json</code>, <code>CONTRIBUTING.md</code>, <code>LICENSE.md</code>, etc) if they don't already exist and will add a webhook to the repository.<br>
```

Comment:
```suggestion
    <li><strong>if the repository is about specifications development</strong>, it needs to be tracked by the <a href="https://labs.w3.org/repo-manager/">W3C repository manager</a>. To do so, someone with admin rights on the repository needs to <a href="https://labs.w3.org/repo-manager/repo/import">import it to the repository manager</a> so contributions can be checked. The repository manager will offer to add some files (<code>w3c.json</code>, <code>CONTRIBUTING.md</code>, <code>LICENSE.md</code>, etc) if they don't already exist and will add a webhook to the repository.<br>
```

Reply:
addressed in https://github.com/w3c/w3c.github.io/pull/96/commits/9adbbf4151d916b4c398cfaf82d69d8c37de4bf4